jolly-jellyfish/src/page_maker/templates/page_maker/example_page.html

```diff
@@ -0,0 +1,15 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<h1><span>Example Page title</span></h1>
+<p>
+    Example Subtitle paragraph
+<p>
+    {% load static %}
+    <img src="{% static "images/placeholder_img.png" %}" alt="Example image">
```

Comment:
This is required to be `<div class="img-wrapper"><img ...></div>` by several of the themes.

Reply:
```suggestion
    <div class="img-wrapper"><img src="{% static "images/placeholder_img.png" %}" alt="Example image"></div>
```